docker-compose.yml

```diff
@@ -19,4 +21,12 @@ services:
     env_file:
       - .env
     expose: ["8080"]
-
+  redis:
```

Comment:
I suppose this is what is replacing django as ~db?

Reply:
No, in Cunliang's setup there was also already a redis server, that acts as a broker/message queue (I think, I'm not a celery expert (yet)). The django db was for storing the different tasks.